### DISCUSSION TITLE
No certificate found matching example.com while having mutlidomain cert *.example.com

### DISCUSSION THREAD

**Discussion by reapercreap** [processone/ejabberd#3693] [Questions] [answered]
Hello,

i have a big problem figuring out why i get this error message. I have requested a multi domain certificate from letsencrypt with certbot and it throws this only for the domain name. The other subdomains like proxy, upload etc. are correctly handled by ejabbed.

Do you know why? Should I create an issue for that? What do you think?

Thank you!

**Comment by licaon-kter** [comment]
So your `hosts:` in .yml is `example.com`?

**Reply by reapercreap** [reply]
no not exactly. i just don't want to post my domain here

**Comment by prefiks** [comment, marked as answer]
Hello,
I don't believe `*.something.com` is valid for `something.com`, you only can use it for `whatever.something.com`. But you can have cert with two subject one for `*.something.com`, and the other one for `something.com` - this should work in your case.

**Reply by reapercreap** [reply]
ohhh! i will try that out. first i used every subdomain.example.com and the normal example.com for the -d option for certbot but the certificate i received only includes example.com and ejabberd gave me the warning for every subdomain that is can not be verified. i thought the *.example.com is valid for the FQDN and the DN

**Reply by reapercreap** [reply]
worked! thank you
